tests/third_party/cupy/core_tests/test_ndarray_reduction.py

```diff
@@ -9,7 +9,6 @@
 from tests.third_party.cupy import testing
 
 
```

Comment:
please refresh `tests/third_party/cupy/creation_tests/test_matrix.py` since it includes new tests for `vander`